### ISSUE TITLE
New metrics to consider from GA4: conversions for stories

### ISSUE THREAD

**Issue by taceybadgerbrook** [wellcomecollection/wellcomecollection.org#8560]
GA4 conversions replaces UA goals. In UA, tracking user progress in reading series sequentially was hampered by the fact that funnels and goals were limited to page urls. As urls did not contain anything pointing to membership of a series, short of creating a funnel and a goal for each url in a series, tracking user progress was not possible.

Given GA4 allows creation of custom events which can be combined in specified sequences to trigger conversions, it's worth investigating whether we can track user progress using partOf in JavaScript code.

**Comment by taceybadgerbrook** [comment]
The main part of this ticket has been achieved by @davidpmccormick 's stories funnel. To close this ticket, do we still need partOf?

**Comment by taceybadgerbrook** [comment]
[Here is a link to James' documentation.](https://www.notion.so/wellcometrust/Prismic-relationships-3f21e5bf9634437088c3af7289537772) From what I understand, Part of returns the Prismic page Id of pages that contain Prismic-generated content. URL formats are:
- /articles/
- /events/
- /exhibitions/
- /guides/
- /pages/
- /projects/
- /series/ (currently used by Editorial to evaluate serials)
- /seasons/

**Comment by taceybadgerbrook** [comment]
@dannybirchall @jennpb @gestchild @davidpmccormick I started using Part of to identify new content on the ever-expanding seasons pages. As this variable only seems to be used in a Looker Studio report I built for Editorial, I think we could drop this and in its place, provide the funnel with page definition /series/.* 

Happy to set up a meeting to demo this with Lalita.

**Comment by dannybirchall** [comment]
@taceybadgerbrook could you demo it to all of us (or at least me) as well please?

**Comment by taceybadgerbrook** [comment]
Happy to. Does your team still do a monthly forum?

From: Danny Birchall ***@***.***>
Date: Tuesday, 21 February 2023 at 14:36
To: wellcomecollection/wellcomecollection.org ***@***.***>
Cc: Tacey Kobayashi ***@***.***>, Mention ***@***.***>
Subject: Re: [wellcomecollection/wellcomecollection.org] New metrics to consider from GA4: conversions for stories (Issue #8560)

@taceybadgerbrook<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftaceybadgerbrook&data=05%7C01%7CT.Kobayashi%40wellcome.org%7C9a994e0c1de84220ac9208db1418f378%7C3b7a675a1fc84983a100cc52b7647737%7C0%7C0%7C638125869624054789%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X8nETabZAgKk8AdHJgJfUiwtQ%2FvHQbuyEDsEG15aSnM%3D&reserved=0> could you demo it to all of us (or at least me) as well please?

**Comment by dannybirchall** [comment]
@taceybadgerbrook better to book ourselves a separate session I reckon.

**Comment by taceybadgerbrook** [comment]
@dannybirchall I've asked Lalita if she would like a demo as she is the one who uses the Data Studio report I built.

**Comment by dannybirchall** [comment]
@taceybadgerbrook all members of my team are going to need access to all the reports that we have

**Comment by taceybadgerbrook** [comment]
@dannybirchall all of your team will have access to GA4 and its reports. 

The report that Lalita has been using uses a custom variable but I think the new GA4 report will meet her needs. The meeting is to check before removing the code that underpins the custom variable.

**Comment by dannybirchall** [comment]
@taceybadgerbrook I understand. There are no reports that are specifically for Lalita though.

**Comment by taceybadgerbrook** [comment]
@dannybirchall [This is the report I built](https://lookerstudio.google.com/u/0/reporting/1MHrOZKl00l0y6lreV0QMvlt4r_zAro94/page/p_ar5e8vn8tc). It's not protected or anything; it's just that Lalita has been using it more than anyone else.

**Comment by pollecuttn** [comment]
Needs meeting between Tacey/Danny/Lalita

**Comment by taceybadgerbrook** [comment]
@gestchild following the meeting I had with Danny and Lalita yesterday, I can confirm that we no longer need to use the variable partOf.

Lalita has not used the Data Studio report I built based on partOf since her evaluation work about a year ago. I demonstrated how GA4 funnels can be set up so that parts of serials can be monitored for sequential reading without using partOf and they seemed happy for this component to be deleted.
